README.md

```diff
@@ -33,14 +33,28 @@ After installing docker, build the docker image:
 docker build -t bot-render . --no-cache=true
 ```
 
-Start a container with the built image:
+## Running the container
+
+There are two ways to run the container locally:
+1. With [Jessie Frazelle seccomp profile](https://github.com/jessfraz/dotfiles/blob/master/etc/docker/seccomp/chrome.json)
+2. With `--cap-add=SYS_ADMIN`
```

Comment:
The seccomp profile is the better way to go.

SYS_ADMIN is for that situation where you don't have user namespace support in your kernel; it's heavy handed and basically a bit of a hammer, but if a user doesn't want to limit the container security wise, they could fall back to this.